### DISCUSSION TITLE
Providing feedback to Gitbook AI via API call

### DISCUSSION THREAD

**Discussion by mukul423** [GitbookIO/community#660] [1. How to]
When interacting with Gitbook AI while asking any question - how can I provide feedback to Gitbook AI via API call for the AI provided response? We are trying to integrate Gitbook AI with our chat client & hence this question as I couldn't find any documentation around this.

**Comment by addisonschultz** [comment]
Hey @mukul423 - are you getting some unexpected results? 

Right now, the only way to give feedback on a result is via the UI within GitBook when using GitBook AI. Let me forward this to the team and we can look into it!

**Reply by mukul423** [reply]
Hi @addisonschultz Nothing to do with wrong answers per se. We just need to have the ability to provide feedback to AI response from our custom chatbot UI which is integrated with Gitbook API. Is there a way to implement this? If not is this "feature" in your near backlog?

**Reply by addisonschultz** [reply]
Hey @mukul423 - I'm not sure I fully understand. What kind of feedback to the AI response do you mean? If you could provide and example that would be helpful 🙏

**Comment by mukul423** [comment]
@addisonschultz I ask the AI - What is Github? AI replies "Github is animal". I "dislike" the answer so that AI knows that this is incorrect. How to provide this feedback of "like/dislike" via API call from our custom chatbot UI (https://gradio.app/)?

**Reply by addisonschultz** [reply]
I see - right now we don't have an exposed endpoint for this functionality unfortunately. But, I'll add some of this context so for us to consider as we continue to build out this part of the API 🙏

**Reply by mukul423** [reply]
In the Slack app integration with Gitbook - how is this aforesaid feedback handled?

**Reply by addisonschultz** [reply]
@mukul423 I don't think there is a way in the Slack integration to collect the feedback you're mentioning? Can you share a photo of what you're referring to?
